src/v0/destinations/klaviyo/transform.js

```diff
@@ -215,45 +218,24 @@ const groupRequestHandler = (message, category, destination) => {
     delete profile.$id;
     profile._id = getFieldValueFromMessage(message, 'userId');
   }
-  const responseArray = [];
 
-  const payload = {
-    profiles: [profile],
+  const subscribeProfile = JSON.parse(JSON.stringify(profile));
```

Comment:
Are trying to clone the `profile` object here ?

Reply:
Ya, we were creating a deep copy of the profile due to previous code changes. Removed it as is not needed now.